plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryThumbnails.php

```diff
@@ -81,7 +81,7 @@ class="wc-block-product-gallery-thumbnails <?php echo esc_attr( $classes_and_sty
 			data-wp-class--wc-block-product-gallery-thumbnails--overflow-right="context.thumbnailsOverflow.right">
 			<div
 				class="wc-block-product-gallery-thumbnails__scrollable"
-				data-wp-init="callbacks.initResizeObserver"
+				data-wp-init="callbacks.initScrollable"
```

Comment:
The `initScrollable` is only a wrapper for other callbacks. We can have multiple init callbacks attached to the same element.
```suggestion
				data-wp-init--resize="callbacks.initResizeObserver"
				data-wp-init--fix-ghost-overflow="callbacks.hideGhostOverflow"
```

Reply:
> The initScrollable is only a wrapper for other callbacks. We can have multiple init callbacks attached to the same element.

Oh nice, I missed this!

I tried with something like this:

```
data-wp-init="callbacks.initResizeObserver"
data-wp-init="callbacks.hideGhostOverflow"
```

but I missed the part they have to be named! Cool tip, thanks!

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/60274/commits/a772c6d519844df6a4d643a70b3fddc4f8a92ca5